### PR TITLE
Handle Exceptions with Empty Dictionary

### PR DESCRIPTION
**Pull request by efe**
https://github.com/Hipo/hipo-drf-exceptions/issues/8

### CODE REVIEW

File: hipo_drf_exceptions/handlers.py [hunk 1]
```diff
@@ -14,7 +14,11 @@ def get_fallback_message(exception):
     elif isinstance(exception, list):
         return get_fallback_message(exception[0])
     elif isinstance(exception, dict):
-        first_key = next(iter(exception))
+        if exception == {}:
+            # A dictionary can be empty. https://github.com/Hipo/hipo-drf-exceptions/issues/8
+            return get_fallback_message("")
+        else:
+            first_key = next(iter(exception))
```
Comment (alicertel) [inline hipo_drf_exceptions/handlers.py]: i think this should be handled in upper if block. If there is a list, it should look for the first non empty one.

I can't think of any case where we should return empty fallback message.

Comment (alicertel) [reply hipo_drf_exceptions/handlers.py]: I mean `elif isinstance(exception, list):` by upper if block.

Comment (efe) [reply hipo_drf_exceptions/handlers.py]: Nice point. What if the list is empty?

Comment (alicertel) [reply hipo_drf_exceptions/handlers.py]: That means there is no exception. There should not be an empty list.

Comment (efe) [reply hipo_drf_exceptions/handlers.py]: OK, We can ignore this edge case. Thank you.

Comment (efe) [reply hipo_drf_exceptions/handlers.py]: @alicertel Ready for re-review.

**Pull request merged**